pkg/util/kubeobjects/configmap/builder.go

```diff
@@ -0,0 +1,28 @@
+package configmap
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/internal/builder"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var setName = builder.SetName[*corev1.ConfigMap]
+var SetNamespace = builder.SetNamespace[*corev1.ConfigMap]
+var SetLabels = builder.SetLabels[*corev1.ConfigMap]
```

Comment:
seems like unused code, why do we need it here 🤔 ?

Reply:
`builder.SetLabels[*corev1.ConfigMap]` is internal

just exported it